rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
+
+//problema 2
+function Get2LargestNo(vect)
+{
+    var numbers = [];
+    numbers[0] = -Infinity;
+    numbers[1] = -Infinity;
+
+    for(i=0; i < vect.length; i++)
+    {
+        if(numbers[1] < vect[i])
+        {
+            numbers[0] = numbers[1];
+            numbers[1] = vect[i];
+        }
+    }
+
+    return numbers;
+}
+
+//problema3
+function ThereAreDuplicate(vect)
+{
+    var suma1 = SumVect(vect);
+	//console.log('suma1 : ' + suma1);
+	//console.log('vect length : ' + vect.length);
+    var suma2 = vect.length * ((vect.length + 1) / 2);
+	//console.log('suma2 : ' + suma2);
+    return !(suma2 == suma1);
+}
+
+function assert( expression, description ) { 
+	var assertStatus = expression ? 'pass' : 'fail'; 
+    console.log(description + ' : ' + assertStatus);
+}; 
+
+
+// unit testing
+var test1 = [1,2,3,4,5,6,7,8,9];
+
+assert(SumVect(test1) == 45, 'Checking the SumVect function');
+
+var MaxNombers = Get2LargestNo(test1);
+assert(MaxNombers[0] == 8, 'Checking the Get2LargestNo function. The precedent largest no. = ' + MaxNombers[0] );
+assert(MaxNombers[1] == 9, 'Checking the Get2LargestNo function. The current largest no. = ' + MaxNombers[1] );
+
+assert(ThereAreDuplicate(test1) == false, 'Checking the ThereAreDuplicate function');
+// -------------------  finnaly tests ----------------------------------------------------------
+
+
+
+//1, 1, 3, 4, 5, 6, 7, 8, 9   false
+//1,2,3,4,5,6,7,8,9,1   false
+//1,1,1,1,1,1,1,1,1,1   false
+//1,2,3,4,5,5,7,8,9,10  false
+//1,2,3,4,5,6,7,8,9   true
+//console.log('Suma elementelor este: ' + SumVect(test1));
+//console.log('Cele mai mari 2 numere sunt: ' + Get2LargestNo(test1));
+//console.log('Exista elemente duplicate: ' + ThereAreDuplicate(test1));
```

Comment:
To keep things easier to follow, you can consider separating each problem in it's on file.